src/test/unit/profile-data.test.js

```diff
@@ -589,7 +589,7 @@ describe('upgrades', function() {
     expect(serializedLhsAsObject).toEqual(serializedRhsAsObject);
   }
   const afterUpgradeReference = unserializeProfileOfArbitraryFormat(
-    require('../fixtures/upgrades/processed-7.json')
```

Comment:
looking closely, I realize `processed-7` was incorrect: the property `preprocessedProfileVersion` was incorrect (still 6) and so was actually "upgraded" by `unserializeProfileOfArbitraryFormat` :/

Fortunately the new one is correct :) But likely we need a last line in the tests, comparing the afterUpgradeReference to itself :) Would you be kind enough to add it ?

Now I see there is no test testing the new code because when upgrading from an old profile, because here `processStartupTime` is 0. I think it would make sense to add a new test with a v7 profile and test upgrading from it, what do you think ?

Same thing to test the adjustement from a gecko v8 profile, as both processes have the same `startTime` in `gecko-8.json`: I think it would make sense to add gecko-v8 profile (maybe simply copy `gecko-v8.json` and update it to change the content process' `startTime` to something slightly larger) and see if the `DOMEvent` marker's timestamp is properly updated.